packages/ethereum-storage/src/config.ts

```diff
@@ -22,22 +22,19 @@ const config: any = {
     defaultNode: {
       host: 'localhost',
       port: 5001,
-      protocol: 'http',
+      protocol: 'http' as StorageTypes.IpfsGatewayProtocol,
       timeout: 30000,
     },
     errorHandling: {
       delayBetweenRetries: 500,
       maxRetries: 3,
     },
-    expectedBootstrapNodes: [
-      '/dns4/ipfs-bootstrap.request.network/tcp/4001/ipfs/QmaSrBXFBaupfeGMTuigswtKtsthbVaSonurjTV967Fdxx',
-
-      '/dns4/ipfs-bootstrap-2.request.network/tcp/4001/ipfs/QmYdcSoVNU1axgSnkRAyHtwsKiSvFHXeVvRonGCAV9LVEj',
-
-      '/dns4/ipfs-2.request.network/tcp/4001/ipfs/QmPBPgTDVjveRu6KjGVMYixkCSgGtVyV8aUe6wGQeLZFVd',
-
-      '/dns4/ipfs-survival.request.network/tcp/4001/ipfs/Qmb6a5DH45k8JwLdLVZUhRhv1rnANpsbXjtsH41esGhNCh',
-    ],
+    expectedBootstrapNodes: {
```

Comment:
the format of the address changed, this accounts for the change while supporting older format